### DISCUSSION TITLE
music assistant for the rest of the family

### DISCUSSION THREAD

**Discussion by userMak** [music-assistant/support#3242] [Q&A] [answered]
Hi all,
although MA is working fine for me, the rest of the family (wife & kids) are not able to make MA to play anything they would need. I am planning to add a tablet or a old phone only for MA and I am wondering if there are any suggestions how to implement this? So looking for a user friendly solution for the rest of the family. (mainly to choose and play radio stations or playlists)
In the tablet or phone should I install HA or is there a way to add MA in browser?
Thanks

**Comment by OzGav** [comment, marked as answer]
Yes you can just login into the HA instance from any device and access MA from there. If you want to reduce the security level you can also expose port 8095 and then access MA directly (if you are using docker then this is how it works normally).

**Comment by userMak** [comment]
yes, this is what I was thinking. But they find MA dashboard it a little confusing. 
The player has small buttons the volume isn't easy to handle (volume up-down) etc . 

1. Is there a way to remove recently played albums, arists, etc and keep only playlists and radio? 
2. Also my playlists are from spotify. is there a way to group them as I wish in MA?
3. I haven't tried it yet but can I add one more Spotify user? And if yes what about the playlists? Are they separate or all mixed together? 

![image](https://github.com/user-attachments/assets/df5010ac-39e1-467f-a22b-46edfdc44db4)

**Reply by OzGav** [reply]
1. Sorry no. The MA UI is not configurable other than the menu items
2. You can find the sorting options in the menu. You would need to work with that and the naming conventions of your playlists in order to group them in some fashion
3. Yes you can add multiple spotify accounts. All playlists are merged into the one library

**Comment by userMak** [comment]
hi again,
is there a way to have as default page when companion app starts the music assistant?

**Reply by OzGav** [reply]
Not that I’m aware but I don’t user the app that much. You should ask this one in the HA forums.

**Reply by OzGav** [reply]
Possible solution? If you expose port 8095 and use an iframe in HA then you could open to a specific view.

**Comment by userMak** [comment]
edit: canceled the latest comment
